build/media-suite_compile.sh

```diff
@@ -1420,7 +1420,7 @@ EOF
     fi
     }
     build_x265
-    log "install" ninja -j1 install
+    log "install" ninja install
```

Comment:
Ninja sometimes tries to install faster than the hard drives/file system can handle. File creation/copy should remain single-threaded when possible.

Reply:
Ah, I just realized that was an install, will fix later.